.github/workflows/fleximod_test.yaml

```
@@ -0,0 +1,22 @@
+on:
+  pull_request:
+    branches: [ development ]
+jobs:
+  fleximod-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # oldest supported and latest supported
+        python-version: ["3.7", "3.x"]
```

I would bump the minimum version up to `3.8` here:

```suggestion
        python-version: ["3.8", "3.x"]
```